SwiftExample/SwiftExampleTests/TIOSwiftTFLiteModelIntegrationTests.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SwiftExampleTests.swift
+//  SwiftExampleTests
+//
+//  Created by Phil Dow on 2/13/19.
+//  Copyright © 2019 doc.ai. All rights reserved.
+//
+
+import XCTest
+import TensorIO
+@testable import SwiftExample
+
+class SwiftExampleTests: XCTestCase {
+
+    let testModelsDir = Bundle.main.path(forResource: "models-tests", ofType: nil)!
+    
+    func bundle(named name: String) -> TIOModelBundle? {
+        let path = URL(fileURLWithPath: testModelsDir).appendingPathComponent(name).path
+        return TIOModelBundle(path: path)
+    }
+    
+    func model(with bundle: TIOModelBundle) -> TIOModel? {
+        guard let model = bundle.newModel() else {
+            print("There was a problem instantiating the model from the bundle")
+            return nil
+        }
+        
+        guard let _ = try? model.load() else {
+            print("There was a problem loading the model")
+            return nil
+        }
+        
+        return model
+    }
+    
+    override func setUp() { }
+    override func tearDown() { }
+
+    // Swift sanity check
+
+    func testMobileNetModel() {
```

Comment:
Like a breath of fresh air in all this Objective C code.

Reply:
as i was writing the swift i was definitely thinking, ahhh, this is nice.